blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.rt

```diff
@@ -0,0 +1,11 @@
+<rt-require dependency="../../ff_module/ff_module-task-event/ff_module-task-event.js" as="TaskEvent"/>
+
+<div class="ff_container-task-event-repeater">
+    <ol class="ff_container-task-event-repeater__items">
+        <li class="ff_container-task-event-repeater__item"
+            key= "event-{eventIndex}"
```

Comment:
Good question. Let's go with `localEventId` for now, and we can change if it turns out that there's something more appropriate.